### PR TITLE
ACRE leader radio copy

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- title
- Allows to copy leader radio settings if within 50m of him. Works only for radios with channels due to public ACRE API limitations.

### CODE REVIEW

File: addons/acre/functions/fnc_copyLeaderSingleRadioChildrenActions.sqf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * 0: Objects <ARRAY>
+ * 1: All <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[bob, ted], false] call afm_main_fnc_example
+ *
+ * Public: No
```
Comment (3Mydlo3) [inline addons/acre/functions/fnc_copyLeaderSingleRadioChildrenActions.sqf:16]: Very interesting description.

File: addons/acre/functions/fnc_copyLeaderRadioChildrenActions.sqf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * Arguments:
+ * 0: Objects <ARRAY>
+ * 1: All <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[bob, ted], false] call afm_main_fnc_example
```
Comment (3Mydlo3) [inline addons/acre/functions/fnc_copyLeaderRadioChildrenActions.sqf:14]:
```suggestion
 * Arguments:
 * 0: Unit to copy radio settings from it's leader <OBJECT>
 *
 * Return Value:
 * List of actions <ARRAY>
 *
 * Example:
 * [player] call afm_acre_fnc_copyLeaderRadioChildrenActions
```

Return value could require more explanation.

File: addons/acre/functions/fnc_getRadioDisplayName.sqf [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/*
+ * Author: veteran29
+ * Returns formatted radio display name with channels/mhZ.
```
Comment (3Mydlo3) [inline addons/acre/functions/fnc_getRadioDisplayName.sqf:4]:
```suggestion
 * Returns formatted radio display name with channels/MHz.
```

**Review by 3Mydlo3 — CHANGES_REQUESTED** [review]

File: addons/acre/functions/fnc_copyLeaderSingleRadioChildrenActions.sqf (hunk 1 above)
Comment (veteran29) [reply addons/acre/functions/fnc_copyLeaderSingleRadioChildrenActions.sqf:16]: Yes

File: addons/acre/functions/fnc_copyLeaderRadioChildrenActions.sqf (hunk 2 above)
Comment (veteran29) [reply addons/acre/functions/fnc_copyLeaderRadioChildrenActions.sqf:14]: It's an internal function for ACE interaction nodes but I'll try to write something more descriptive.

**Review by 3Mydlo3 — APPROVED** [review]
It will be really useful feature for our newcomers.

**Pull request merged**